apps/internal/local/server_test.go

```diff
@@ -111,6 +112,15 @@ func TestServer(t *testing.T) {
 			statusCode:   200,
 			testTemplate: true,
 		},
+		{
+			desc:              "Error: Query Values missing 'state' key, using default fail error page - XSS test",
+			reqState:          "state",
+			port:              0,
+			q:                 url.Values{"error": []string{"<script>alert('this code snippet was executed')</script>"}, "error_description": []string{"error_description"}},
```

Comment:
Thanks. Please also add the similar scripts to the "error_description" and have it tested, if it has not been covered by any existing test case.

Reply:
Hi @rayluo , I have added some more test cases. (please see my last two commits).
1: Updating more test case for default fail page.
2: Added two more test cases for the SDK _user_ provided option error page (one for each html template field).